test/crunch++/meson.build

```diff
@@ -1,7 +1,17 @@
-libCrunchppTestsNorm = ['testArgsParser', 'testCrunch++', 'testBad', 'testRegistration', 'testLogger']
+libCrunchppTestsNorm = ['testCrunch++', 'testBad', 'testRegistration', 'testLogger']
 libCrunchppTestsExcept = ['testTester']
 libCrunchppTests = libCrunchppTestsNorm + libCrunchppTestsExcept
 
+if cxx.get_id() == 'msvc' or cxx.version().version_compare('< 19.30.30704')
+	# MSVC 2019 crashes on testArgsParser
+	# https://developercommunity.visualstudio.com/t/Crash-on-constexpr-creation-of-a-std::ar/1547919
+	warning('Skipping testArgsParser test as it crashes MSVC')
+else
+	libCrunchppTestsNorm += ['testArgsParser']
+endif
+
+message('MSVC version: @0@'.format(cxx.version()))
```

Comment:
this should probably be gated behind `if isMSVC` (and that constant propagated through to here).

Reply:
I had that change done as part of the MSYS fix, so I'll backport it now.